webpack.config.js

```diff
@@ -40,11 +40,11 @@ module.exports = {
         directory: './src',
       },
       {
-        publicPath: '/aspera-browser-sdk-js/docs',
+        publicPath: '/aspera-sdk-js/docs',
         directory: './dist/js/docs',
       },
     ],
-    host: 'browser-sdk.aspera.us',
+    host: 'js-sdk.aspera.us',
```

Comment:
```suggestion
    host: 'sdk.aspera.us',
```
Not quite what this is...

Reply:
This is for internal dev running. Just needs to be a little unique as we use this for other JavaScript projects like http gateway and aoc stuff.